buildbot/master/master.cfg

```diff
@@ -242,6 +242,36 @@ gonk_factory = create_servo_factory([
     steps.ShellCommand(command=["bash", "./etc/ci/manifest_changed.sh"], env=gonk_compile_env),
 ])
 
+arm32_compile_env = dict({'PATH': '/home/servo/bin:$PATH',
+                          'BUILD_TARGET': 'arm-unknown-linux-gnueabihf',
+                          'TRIPLET': 'arm-linux-gnueabihf',
```

Comment:
Is the `TRIPLET` variable required at run time or is it only used to set `CC` and `CXX`? If the latter, I'd prefer to use python string formatting to bake the right value into those variables and avoid polluting the environment with the `TRIPLET` variable.